Userland/Libraries/LibWeb/CSS/StyleValue.h

```diff
@@ -814,6 +815,40 @@ class BorderRadiusStyleValue final : public StyleValue {
     Length m_vertical_radius;
 };
 
+class CombinedBorderRadiusStyleValue final : public StyleValue {
```

Comment:
Is the purpose of this class simply to achieve space separation (instead of comma separation) in the `to_string()` output? If so, it seems like we could make that a flag of `StyleValueList` instead(?)

Reply:
It's not that simple, unfortunately. The syntax for `border-radius` with elliptical radii is: `border-radius: x1 x2 x3 x4 / y1 y2 y3 y4`, so a StyleValueList doesn't work. :^(

IMO this is the best way to handle it. I'd just add that the parser code should also produce one of these instead of a StyleValueList.

Reply:
Additionally, the CSS property order is not intuitive for everyone (top left, top right, bottom right, bottom left), so it's better to have direct unambiguous accessors instead of doing indexing and hoping that all the code indexes identically.